docs/upgrade/11_to_12/workflows_mariadb.py

```diff
@@ -233,12 +235,20 @@ def parse_bool(value):
         workflow_operations = []
         workflow_operation_config = []
         workflow_config = []
+        description_limited = get_node_value(root, 'description', WORKFLOW_NS)
+        if description_limited is None:
+          description_limited = None
+        else:
+          description_limited = description_limited.lstrip().rstrip()
+          description_limited = description_limited.replace('\n','').replace('    ', ' ')
```

Comment:
Replacing newline chars may not be desirable for everyone?

Reply:
Its desirable for us.

Reply:
Fair enough. Arguably shouldn't bother too many users either way.